reference/docs-conceptual/dev-cross-plat/vscode/using-vscode-for-debugging-compiled-cmdlets.md

```diff
@@ -0,0 +1,204 @@
+   array, you see the `configuration` picker. If you don't see this list, select
+   **Add Configuration**.
+
+1. To create a default debug configuration, select **Launch .NET Core Console App**:
+
+   ![Launch .NET Core Console App](media/using-vscode-for-debugging-compiled-cmdlets/add-configuration-dialog.png)
+
+1. Edit the `name`, `program`, `args`, and `console` fields as follows:
+
+   ```json
+    {
+        "name": "PowerShell cmdlets: pwsh",
+        "type": "coreclr",
+        "request": "launch",
+        "preLaunchTask": "build",
+        "program": "pwsh",
+        "args": [
+            "-NoExit",
+            "-NoProfile",
+            "-Command",
+            "Import-Module ${workspaceFolder}/myModule/bin/Debug/netstandard2.0/myModule.dll",
+        ],
+        "cwd": "${workspaceFolder}",
+        "stopAtEntry": false,
+        "console": "integratedTerminal"
+    }
+   ```
+
+The `program` field is used to launch `pwsh` so that the cmdlet being debugged can be run. The
+`-NoExit` argument prevents the PowerShell session from exiting as soon as the module is imported.
+The path in the `Import-Module` argument is the default build output path when you've followed the
+[Writing Portable Modules][] guide. If you've created a module manifest (`.psd1` file), you should
+use the path to that instead. The `/` path separator works on Windows, Linux, and macOS. You must
+use the integrated terminal to run the PowerShell commands you want to debug.
+
+### Configuring launch.json for Windows PowerShell
+
+This launch configuration works for testing your cmdlets in Windows PowerShell (`powershell.exe`).
+Create a second launch configuration with the following changes:
+
+1. `name` should be `PowerShell cmdlets: powershell`
+
+1. `type` should be `clr`
+
+1. `program` should be `powershell`
+
+   It should look like this:
+
+   ```json
+    {
+        "name": "PowerShell cmdlets: powershell",
+        "type": "clr",
+        "request": "launch",
+        "preLaunchTask": "build",
+        "program": "powershell",
+        "args": [
+            "-NoExit",
+            "-NoProfile",
+            "-Command",
+            "Import-Module ${workspaceFolder}/myModule/bin/Debug/netstandard2.0/myModule.dll",
+        ],
+        "cwd": "${workspaceFolder}",
+        "stopAtEntry": false,
+        "console": "integratedTerminal"
+    }
+   ```
+
+## Launching a debugging session
+
+Now everything is ready to begin debugging.
+
+- Place a breakpoint in the source code for the cmdlet you want to debug:
+
+  ![A breakpoint shows as a red dot in the gutter](media/using-vscode-for-debugging-compiled-cmdlets/set-breakpoint.png)
+
+- Ensure that the relevant **PowerShell cmdlets** configuration is selected in the configuration
+  drop-down menu in the **Debug** view:
+
+  ![Select the launch configuration](media/using-vscode-for-debugging-compiled-cmdlets/select-launch-configuration.png)
+
+- Press <kbd>F5</kbd> or click on the **Start Debugging** button
```

Comment:
Is F5 the shortcut for all OSes?  Since we mention this works on Windows, Linux, macOS, we should provide shortcuts for each on this step.

Reply:
It is.